src/cloud/audiocom/LinkAccountDialog.cpp

```diff
@@ -0,0 +1,117 @@
+/*  SPDX-License-Identifier: GPL-2.0-or-later */
+/*!********************************************************************
+
+  Audacity: A Digital Audio Editor
+
+  LinkAccountDialog.cpp
+
+  Dmitry Vedenko
+
+**********************************************************************/
+#include "LinkAccountDialog.h"
+
+#include <wx/button.h>
+#include <wx/textctrl.h>
+#include <wx/statline.h>
+#include <wx/weakref.h>
+
+#include "ShuttleGui.h"
+
+#include "BasicUI.h"
+#include "CodeConversions.h"
+
+#include "OAuthService.h"
+
+#include "LinkFailedDialog.h"
+
+namespace cloud::audiocom
+{
+LinkAccountDialog::LinkAccountDialog(wxWindow* parent)
+    : wxDialogWrapper(
+         parent, wxID_ANY, XO("Link account"), wxDefaultPosition, { 480, -1 },
+         wxDEFAULT_DIALOG_STYLE)
+{
+   ShuttleGui s(this, eIsCreating);
+
+   s.StartVerticalLay();
+   {
+      s.StartInvisiblePanel(16);
+      {
+         s.SetBorder(0);
+         s.AddFixedText(XO("Enter token to link your account"));
+
+         s.AddSpace(0, 4, 0);
+
+         mToken = s.AddTextBox(TranslatableString {}, {}, 60);
+         mToken->SetName(XO("Token").Translation());
```

Comment:
You understand that `SetName` affects audible screen reader output?

Reply:
Yes, that is the reason for SetName here